### PR TITLE
Polish many little things

### PR DESCRIPTION
**Pull request by jasmussen**
This PR bundles a bunch of smaller tweaks together into one. All of these tweaks are primarily visual, though.

1. Tune links —  change the link color, make code boundary work
2. Sped up the appear animation. 
3. Polished the quote style. It was off.
4. Normalized all captions.
5. Normalized all placeholders.

### CODE REVIEW

File: blocks/components/editable/style.scss [hunk 1]
```diff
@@ -13,20 +13,42 @@
...
+		box-shadow: 0 0 0 1px $blue-medium-100;
+		background: $blue-medium-100;
+	}
+
+	code {
+		padding: 0 2px;
+		margin: 0 -2px;
+		border-radius: 2px;
+		color: $dark-gray-800;
+		background: $light-gray-100;
```
Comment (iseulde) [inline blocks/components/editable/style.scss]: I'm finding this quite hard to see.

Comment (iseulde) [inline blocks/components/editable/style.scss]: I also have the feeling that the spaces around it are quite small. The margins look great imho for selection, but not sure about the normal background.

File: blocks/components/editable/style.scss [hunk 2]
```diff
@@ -13,20 +13,42 @@
...
 }
 
 figcaption.blocks-editable {
 	margin-top: 0.5em;
 	color: $dark-gray-100;
 	text-align: center;
 	font-size: $default-font-size;
+
+	> p {
+		font-size: $default-font-size;
```
Comment (iseulde) [inline blocks/components/editable/style.scss]: I think we can remove this now?

Comment (iseulde) [inline blocks/components/editable/style.scss]: (No `p`s in `figcaption`.)

Comment (jasmussen) [inline blocks/components/editable/style.scss]: Are we sure there will never be paragraphs in any captions? If yes, I'll remove it.

File: blocks/components/editable/style.scss [hunk 3]
```diff
@@ -69,3 +91,44 @@ input.editable-format-toolbar__link-input {
...
+}
+
+.placeholder__label {
+	display: flex;
+	justify-content: center;
+	font-weight: bold;
+	margin-bottom: 1em;
+
+	.dashicon {
+		margin-right: 1ch;
```
Comment (iseulde) [inline blocks/components/editable/style.scss]: I had never heard of this before :)

File: blocks/components/editable/style.scss (hunk 2 above)
Comment (iseulde) [inline blocks/components/editable/style.scss]: Pretty sure :)

File: blocks/components/editable/style.scss (hunk 1 above)
Comment (jasmussen) [inline blocks/components/editable/style.scss]: Can you clarify? Do you mean the light gray is too light? or the contrast between the text inside the gray bg is too low?

File: blocks/components/editable/style.scss (hunk 3 above)
Comment (jasmussen) [inline blocks/components/editable/style.scss]: Yep, me either. I think @BE-Webdesign introduced me to it!

File: blocks/components/editable/style.scss (hunk 1 above)
Comment (iseulde) [inline blocks/components/editable/style.scss]: So when nothing is selected, I can barely see the background on the `code` tag.

**Comment by jasmussen** [comment]
Darkened the code tag, and removed the nested P style!

**Review by ellatrix — APPROVED** [review]

**Comment by iseulde** [comment]
Yay! Still finding it a bit light, but maybe I'm just getting tired :D

**Comment by jasmussen** [comment]
Thank you!

We can try this for a bit, and darken if need be.

File: blocks/components/editable/style.scss [hunk 4]
```diff
@@ -69,3 +87,44 @@ input.editable-format-toolbar__link-input {
...
+	justify-content: center;
+	padding: 1em;
+	min-height: 200px;
+	text-align: center;
+	font-family: $default-font;
+	font-size: $default-font-size;
+	background: $light-gray-100;
+}
+
+.placeholder__label {
```
Comment (aduth) [inline blocks/components/editable/style.scss]: What's the plan for this? Is this going to become a general-purpose component? We're breaking our own self-imposed CSS naming guidelines with this change:

https://github.com/WordPress/gutenberg/blob/master/docs/coding-guidelines.md#naming

File: blocks/components/editable/style.scss [hunk 5]
```diff
@@ -13,13 +13,31 @@
...
+	}
+
+	code {
+		padding: 0 2px;
+		margin: 0 -2px;
+		border-radius: 2px;
+		color: $dark-gray-800;
+		background: $light-gray-200;
+		font-family: $editor-html-font;
+		font-size: 14px;
```
Comment (aduth) [inline blocks/components/editable/style.scss]: Should we be so explicit about font size for code fragments? Should this be specific to only certain blocks instead? Currently it applies to e.g. Heading blocks, making for some odd visual appearance:

image

Comment (jasmussen) [reply blocks/components/editable/style.scss]: Good thought. Fixed in https://github.com/WordPress/gutenberg/pull/8914.

**Pull request merged**